unixfs.go

```diff
@@ -28,16 +28,6 @@ var (
 	ErrUnrecognizedType    = errors.New("unrecognized node type")
 )
 
-// FromBytes unmarshals a byte slice as protobuf Data.
-func FromBytes(data []byte) (*pb.Data, error) {
```

Comment:
So, let's restore this function as mentioned above and add a deprecation notice:

```
// Deprecated: Use `FSNodeFromBytes` instead to avoid direct manipulation of `pb.Data.format`.
```

Reply:
OK, I had added the `Deprecated ` description.